### ISSUE TITLE
Swagger webjars don't work

### ISSUE THREAD

**Issue by YaroslavYakymenko** [javalin/javalin-openapi#103]
Version of javalin-openapi-plugin and javalin-swagger-plugin is **5.1.4**
I have the following OpenAPI and swagger config:

```
      OpenApiConfiguration openApiConfiguration = new OpenApiConfiguration();
      openApiConfiguration.setInfo(applicationInfo);
      openApiConfiguration.setSecurity(new SecurityConfiguration()
          .withSecurityScheme(OPENAPI_IAM_KEY, new OAuth2(OPENAPI_IAM_KEY)
              .withFlow(oAuth2Flow))
          .withSecurity(new Security(OPENAPI_IAM_KEY)
              .withScope("openid")
              .withScope("roles")));
      javalinConfig.plugins.register(new OpenApiPlugin(openApiConfiguration));
      
     javalinConfig.plugins.register(new SwaggerPlugin(new SwaggerConfiguration()));
```

Swagger UI doesn't work. It tries to load swagger webjars by https://webjars/... instead of https://host/webjars/...

<img width="669" alt="image" src="https://user-images.githubusercontent.com/23578698/202515447-7a982930-b948-4e57-9d37-e729ae4c8496.png">

**Comment by dzikoysk** [comment]
Do you use custom context path for routing plugin?

**Comment by YaroslavYakymenko** [comment]
No, should I?

**Comment by dzikoysk** [comment]
No, but this is related to the fact I've added support for it in 5.1.4. You can downgrade to 5.1.3 for now and I'll take a look on that in the evening.

**Comment by dzikoysk** [comment]
It's fixed in `5.1.4-1`, should be available in Maven Central up to an hour.
